packages/common/src/invitationCode.ts

```diff
@@ -5,6 +5,7 @@ import Logger from './logger'
 const logger = Logger('invite')
 
 export const PSK_PARAM_KEY = 'k'
+export const OWNER_ORBIT_DB_IDENTITY_PARAM_KEY = 'o'
```

Comment:
I'm afraid this indicates another breaking change, doesn't it?
If that's true then we should reconsider not implementing this round of work in the first release of v2